mmv1/products/healthcare/terraform.yaml

```diff
@@ -82,6 +82,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           dataset_name: "example-dataset"
           dicom_store_name: "example-dicom-store"
           pubsub_topic: "dicom-notifications"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "healthcare_dicom_store_bq_stream"
```

Comment:
this section needs to have `min_version: beta` as well; currently, the test is being generated for GA (but will never pass because the field is beta-only.)